mono/metadata/loader.c

```diff
@@ -1187,6 +1187,75 @@ is_absolute_path (const char *path)
 	return g_path_is_absolute (path);
 }
 
+typedef enum MonoLookupPInvokeErr {
+	LOOKUP_PINVOKE_ERR_OK = 0, /* No error */
```

Comment:
This could just be typedef enum {.
Also, internal structures/functions can have a Mono prefix, but its not required.